pandas/tests/groupby/test_skew.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+import pandas as pd
+import pandas._testing as tm
+
+
+def test_groupby_skew_equivalence():
```

Comment:
Do we do this for the other functions? I like the idea; seems like it would be good to consolidate this with others in a follow up